po/es.po

```diff
@@ -0,0 +1,706 @@
+
+#: src/install.rs:996 src/install.rs:1941
+msgid "{}-{} is up to date -- skipping"
+msgstr "{}-{} esta actualizado -- saltando"
+
+#: src/install.rs:1011
+msgid "Calculating conflicts..."
+msgstr "Calculando conflictos..."
+
+#: src/install.rs:1017
+msgid "Calculating inner conflicts..."
+msgstr "Calculando conflictos internos..."
+
+#: src/install.rs:1029
+msgid "Inner conflicts found:"
+msgstr "Conflictos internos encontrados:"
+
+#: src/install.rs:1051
+msgid "Conflicts found:"
+msgstr "Conflictos encontrados:"
+
+#: src/install.rs:1074
+msgid "Conflicting packages will have to be confirmed manually"
+msgstr "Los paquetes en conflicto tendrán que confirmase manualmente"
+
+#: src/install.rs:1078
+msgid "can not install conflicting packages with --noconfirm"
+msgstr "no se pueden instalar los paquetes en conflicto con --noconfirm"
+
+#: src/install.rs:1117
+msgid "Repo"
+msgstr "Repo"
+
+#: src/install.rs:1124
+msgid "Repo Make"
+msgstr "Constructor de Make"
+
+#: src/install.rs:1138
+msgid "Aur Make"
+msgstr "Constructor de AUR"
+
+#: src/install.rs:1240
+msgid "fetching devel info..."
+msgstr "obteniendo info de desarrollo...."
+
+#: src/install.rs:1320
+msgid "Signing packages..."
+msgstr "Firmando paquetes..."
+
+#: src/install.rs:1398 src/install.rs:1403
+msgid "failed to download sources for '{}'"
+msgstr "error al descargar fuentes para '{}'"
+
+#: src/install.rs:1408 src/install.rs:1447 src/install.rs:1455
+msgid "failed to build '{}'"
+msgstr "error al construir '{}'"
+
+#: src/install.rs:1411
+msgid "{}: parsing pkg list..."
+msgstr "{}: parseando lista pkg..."
+
+#: src/install.rs:1473
+msgid "adding {} to the install list"
+msgstr "añadiendo {} a la lista de instalación"
+
+#: src/install.rs:1532
+msgid "could not find package '{:pkg}' in package list for '{:base}'"
+msgstr "no se pudo encontrar paquete '{:pkg}' en lista de instalación para '{:base}'"
+
+#: src/install.rs:1670
+msgid "can't find package name in packagelist: {}"
+msgstr "no se puede encontrar el nombre del paquete en lista de paquetes: {}"
+
+#: src/install.rs:1743
+msgid "There are {} members in group"
+msgstr "Hay {} miembros en el grupo"
+
+#: src/install.rs:1768
+msgid ""
+"\n"
+"\n"
+"Enter a selection (default=all): "
+msgstr ""
+"\n"
+"\n"
+"Introduce una elección (default=all): "
+
+#: src/install.rs:1965
+msgid "{}-{} is up to date -- skipping install"
+msgstr "{}-{} esta actualizado -- saltando instalación"
+
+#: src/keys.rs:47
+msgid "keys need to be imported:)"
+msgstr "claves necesitan importarse :)"
+
+#: src/keys.rs:52
+msgid "     {:key} wanted by: {:base}"
+msgstr "     {:key} requerido por: {:base}"
+
+#: src/news.rs:57
+msgid "No Date "
+msgstr "Sin Fecha "
+
+#: src/news.rs:60
+msgid "No Title"
+msgstr "Sin Título"
+
+#: src/news.rs:69
+msgid "no new news"
+msgstr "no hay noticias"
+
+#: src/query.rs:105
+msgid " [ignored]"
+msgstr " [ignorado]"
+
+#: src/repo.rs:164
+msgid "failed to get current exe"
+msgstr "error al obtener exe actual"
```

Comment:
```suggestion
msgstr "error al obtener binario actual"
```

Reply:
This exe is referering to a binary or an executable? executable would be better translated as "ejecutable" rather than "binario"

Reply:
binary and executable are the same thing really.

Reply:
Yes, executable would be translated as ejecutable imo as well, but since it says exe I went with binary. Either way, I think exe should be replaced because it requires the reader to know the English spelling of executable, which defeats the purpose of translations.